login-workflow/src/new-architecture/screens/VerifyCodeScreen/VerifyCodeScreen.tsx

```diff
@@ -21,18 +23,25 @@ export const VerifyCodeScreen: React.FC<VerifyCodeScreenProps> = (props) => {
     const { t } = useLanguageLocale();
     const regWorkflow = useRegistrationWorkflowContext();
     const { actions } = useRegistrationContext();
+    const errorConfig = useErrorContext();
     const { nextScreen, previousScreen, screenData, currentScreen, totalScreens } = regWorkflow;
     const { emailAddress } = screenData.CreateAccount;
 
     const [verifyCode, setVerifyCode] = useState(screenData.VerifyCode.code);
     const [isLoading, setIsLoading] = useState(false);
+    const [error, setError] = useState<AuthError>({ cause: { title: '', errorMessage: '' } });
```

Comment:
You could write a new hook (as part of the Error provider) that you can share between components so you don't have to set up this local state every time. Usage would look something like:

```
// hook would maintain the internal state for the error and handle the parsing of the error object that gets thrown
// ErrorManager component would handle parsing the config parameters
// then you only have to define these behaviors in one place
const { triggerError, ErrorManager } = useErrorManagement();
...

try{
    makeApiCall();
}
catch(error){
    triggerError(error);
}
...
<ErrorManager>{children}<ErrorManager>
```